UPGRADE-1.14.md

```diff
@@ -0,0 +1,3 @@
+# UPGRADE FROM `v1.13.X` TO `v1.14.0`
+
+1. The class `Sylius\Bundle\CoreBundle\Twig\StateMachineExtension` has been deprecated and will be removed in Sylius 2.0. Use `Sylius\Abstraction\StateMachine\Twig\StateMachineExtension` instead.
```

Comment:
Noting the twig functions affected by this would be nice

Reply:
The `Sylius\Bundle\CoreBundle\Twig\StateMachineExtension` has never been registered.